ghsa/ghsa.go

```diff
@@ -102,6 +103,16 @@ func (c Config) update(ecosystem SecurityAdvisoryEcosystem) error {
 		}
 		ghsa.Package.Name = strings.TrimSpace(ghsa.Package.Name)
 
+		// Part Swift packages have `https://` prefix or `.git` suffix
+		// e.g. https://github.com/github/advisory-database/blob/76f65b0d0fdac39c8b0e834ab03562b5f80d5b27/advisories/github-reviewed/2023/06/GHSA-r6ww-5963-7r95/GHSA-r6ww-5963-7r95.json#L21
+		// https://github.com/github/advisory-database/blob/76f65b0d0fdac39c8b0e834ab03562b5f80d5b27/advisories/github-reviewed/2023/07/GHSA-jq43-q8mx-r7mq/GHSA-jq43-q8mx-r7mq.json#L21
+		// Trim them to fit the same format
+		// as in https://github.com/github/advisory-database/blob/76f65b0d0fdac39c8b0e834ab03562b5f80d5b27/advisories/github-reviewed/2023/06/GHSA-qvxg-wjxc-r4gg/GHSA-qvxg-wjxc-r4gg.json#L21
+		if ecosystem == Swift {
+			ghsa.Package.Name = strings.TrimLeft(ghsa.Package.Name, "https://")
+			ghsa.Package.Name = strings.TrimRight(ghsa.Package.Name, ".git")
+		}
+
```

Comment:
Can we trim them in trivy-db rather than vuln-list-update? I'd keep the original data as much as possible.

Reply:
Now i only update the directory names (https://github.com/aquasecurity/vuln-list-update/pull/233/commits/61450fa248c090cfee544c86783fcfa4af12559f).
Names in JSON are stored as in GHSA.
https://github.com/aquasecurity/vuln-list-update/blob/7bb79eeb598d5a580923c5107b3fe5abd9874702/ghsa/testdata/swift/github.com/grpc/grpc-swift/GHSA-r6ww-5963-7r95.json#L6